build_scripts/build_usd.py

```diff
@@ -248,7 +249,7 @@ def GetCPUCount():
     except NotImplementedError:
         return 1
 
-def Run(cmd, logCommandOutput = True):
+def Run(cmd, logCommandOutput = True, envOverride = None):
```

Comment:
does this envOverride ever get set to anything but None? If no, could I request that that we don't introduce a new parameter. build_usd needs less features, not more, and adding an override seems to add a vector for new complexity down the line.

Reply:
In this diff, it seems to be either set to None, or passed from function to function, but not uniquely set.

Reply:
In RunCmake it is passed into run in the CMake invocation, and then used to build OpenSubdiv - the parts are always moving and I immediately noticed it could be removed from InstallOpenEXR and possibly it can be removed from Opensubdiv by now, I will give that a try

Reply:
thanks :)